### ISSUE TITLE
New webman kills licenses?

### ISSUE THREAD

**Issue by superretro2018** [aldostools/webMAN-MOD#1059]
I use webman emulators and when launching a psp game with an emulator it works but when i exit the game my ps3 licenses get revoked and i have to use the cbomb fix option in cfw to fix the issue everytime. heres the error code
![IMG_4559](https://github.com/aldostools/webMAN-MOD/assets/58452964/9750fcb4-b9ea-47e7-bf2b-7c7c84b9384c)

**Comment by aldostools** [comment]
It looks like you're experiencing what I warned in this post
https://twitter.com/aldostools/status/1777788927740444789
![image](https://github.com/aldostools/webMAN-MOD/assets/974897/b7f987d6-dde6-4ee6-b9b6-a0b011bce7e2)

It's an issue in Cobra, not in webMAN MOD.

**Comment by superretro2018** [comment]
Is it fixable in the future?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by aldostools** [comment]
I don't think so. I has been like that since Cobra was released open source 10 years ago.

The solution is reinstall the RAP for the game or re-download the RIF license from the PSN Store. The RIF is installed when you start the download of the game. Don't need complete the download. You can cancel it.

**Comment by superretro2018** [comment]
ok thanks

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by superretro2018** [comment]
regardless the cbomb fix lets me fix my licenses everytime it happens so its not that big of a deal

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by aldostools** [comment]
Then it's an issue with the clock of your PS3... using update date/time from XMB Settings should fix the issue too.

**Comment by superretro2018** [comment]
oh ok thanks

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by superretro2018** [comment]
i am having an issue with ps2 emulator and memory cards. my ps2 game is not detecting my created memory card that you normally create is this a bug?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by aldostools** [comment]
It's a layer 8 issue. There are different kind of PS2 saves used by PS2 ISO or discs and by the PS2 Classics (encrypted).
Each type is created and configured in a different way.

Make sure that you're using the correct save. Apollo Save Tool can manage both types of saves.

More technical information is found at https://psdevwiki.com/ps3/PS2_Savedata
